docs/getting-started.md

```diff
@@ -1,21 +1,33 @@
 ---
 title: Getting started
-custom_edit_url: https://github.com/stryker-mutator/stryker-net/edit/master/docs/Getting-started.md
+custom_edit_url: https://github.com/stryker-mutator/stryker-net/edit/master/docs/getting-started.md
 ---
+# Install prerequisites
 
-# 1 Install
+## General
 
-Stryker.NET can both be installed globally and locally.
+Stryker is installed using the dotnet cli as a [dotnet tool](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools)
+
+Stryker requires dotnet 5.0 or newer: [Install dotnet](https://dotnet.microsoft.com/download)
+
+## Dotnet framework specific
+Nuget is required for dotnet framework. Follow the instructions at [Install nuget](https://docs.microsoft.com/en-us/nuget/install-nuget-client-tools#windows) including adding nuget to the path. Otherwise stryker will not be able to find and use nuget.
+
+Make sure NuGet targets and build tasks are installed. You can install them with visual studio by going to Tools >  Get Tools and Features > Individual components > Code tools
+
+[NuGet targets and build tasks feature](./images/install-nuget-targets.png)
+
+# Install stryker
+
+Stryker can be installed globally for your whole system or locally for a specific project.
 
 ### Install globally
 ```bash
 dotnet tool install -g dotnet-stryker
 ```
 
 ### Install in project
-Dotnet tools can also be installed on a project level. This requires the following steps:
-
-Create a file called dotnet-tools.json in your project folder, if this is your first local tool.
+Create a file called dotnet-tools.json in your project folder if this is your first local tool.
```

Comment:
`dotnet new tool-manifest` will [also work](https://docs.microsoft.com/en-us/dotnet/core/tools/local-tools-how-to-use).

Reply:
Look two lines down ;)

Reply:
Okay yes sorry :D